### PR TITLE
Rg/62 tech debt general UI tweaks before the next sprint

### PR DESCRIPTION
**Pull request by ranegray**
updated ui across signin, signup, and interview pages.

### CODE REVIEW

File: client/src/components/DashboardNavBar.jsx
Comment (jordan-carlson99) [inline client/src/components/DashboardNavBar.jsx:1]: Where did this go/was this done somewhere else?

**Review by jordan-carlson99 — COMMENTED** [review]
Looks good, I wanna test that react container thing before approving

File: client/src/components/InterviewCard/InterviewCard.jsx [hunk 1]
```diff
@@ -54,17 +53,24 @@ const InterviewCard = () => {
     setNote(question.note);
   };
 
+  const copyLink = () => {
+    navigator.clipboard.writeText(`${window.location.origin}/studentinterview`);
+    toast("Copied link to clipboard!", {
```
[the remark this answers is not in the document]
Comment (jordan-carlson99) [reply client/src/components/InterviewCard/InterviewCard.jsx:58]: pulled and tested it, looks good approving

**Review by jordan-carlson99 — DISMISSED** [review]

File: client/package.json
Comment (joegor85) [inline client/package.json:1]: I wondered why you removed the sandpack themes until I saw it moved above to line 24. Cool.

File: client/src/components/DashboardNavBar.jsx
Comment (joegor85) [reply client/src/components/DashboardNavBar.jsx:1]: Yeah, I think this was mostly replaced by the SidebarNav.jsx file

**Review by joegor85 — APPROVED** [review]
Looks good as long as we fix the one small issue with the test.

File: client/src/components/DashboardNavBar.jsx
Comment (ranegray) [reply client/src/components/DashboardNavBar.jsx:1]: Yeah it was replaced.

**Review by joegor85 — APPROVED** [review]
Awesome.

**Pull request merged**